spec/requests/api/netflix_api_responds_with_top_10_movies_globally_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe 'GET /api/movies', type: :request do
+  let(:top_100_response) do
+    file_fixture('top_movie_search_api_response.json').read
+  end
+
+  let(:filtered_top_10) do
+    file_fixture('top_10_avgrating_filter.json').read
+  end
+
+  describe 'successfully receives top 100 movies globally' do
+    before do
+      stub_request(:get, 'https://unogsng.p.rapidapi.com/search?type=movie&orderby=rating')
+        .to_return(status: 200, body: top_100_response)
+      get '/api/movies'
+    end
+
+    it 'is expected to respond 200' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'filter top 10 movies globally' do
+      expect(response_json['body'].count).to eq 10
+    end
+
+    it 'is expected to return filter low Netflix rating movies' do
+      expect(response_json['body']).to eq JSON.parse(filtered_top_10)
+    end
+  end
+end
```

Comment:
Missing the sad path ☔

Reply:
Can you give an idea of what a sad path should be?)) we were thinking about it but user cannot do anything at that point to break the call as everything is hard coded in the API. We thought that when we will add taking params from client we will add it. But now we dont know what to do there

Reply:
What should happen if you don't get any response from the external API? ;)

Reply:
We added the sad path. Test is written only for 500 error, but we pass to client whatever error Netflix API is giving us. So we think this will handle all errors.